Makefile

```diff
@@ -56,7 +56,8 @@ ifeq ($(VERBOSE), false)
   VERBOSE_TESTS_FLAGS:=
 endif
 ifeq ($(COVERAGE), true)
-  EXTRA_ARGS+= -test.coverprofile=../functionals.cover
+  TEST_COVERPROFILE?=../functionals.cover
```

Comment:
please don't change that, we don't need that

Reply:
Notice that the original `run-k8s-tests.sh` renamed the `functionals.cover` - rather I gave the option to set the wanted target name.